pkgs/desktops/deepin/dtkcore/default.nix

```diff
@@ -3,41 +3,39 @@
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "dtkcore";
-  version = "2.0.10";
+  version = "2.0.12.1";
 
   src = fetchFromGitHub {
     owner = "linuxdeepin";
     repo = pname;
     rev = version;
-    sha256 = "0dwpq6c38gaa95mgjnwj3vjz57n0cz6jfk950xi6s9ww2f4g6kq7";
+    sha256 = "1akfzkdhgsndm6rlr7snhpznxj0w351v6rr8vvnr6ka2dw75xsl4";
   };
 
   nativeBuildInputs = [
     pkgconfig
     qmake
     pythonPackages.wrapPython
+    deepin.setupHook
   ];
 
   buildInputs = [
     gsettings-qt
   ];
 
   postPatch = ''
-    # Only define QT_HOST_DATA if it is empty
-    sed '/QT_HOST_DATA=/a }' -i src/dtk_module.prf
-    sed '/QT_HOST_DATA=/i isEmpty(QT_HOST_DATA) {' -i src/dtk_module.prf
+    searchHardCodedPaths  # debugging
 
     # Fix shebang
     sed -i tools/script/dtk-translate.py -e "s,#!env,#!/usr/bin/env,"
   '';
 
-  preConfigure = ''
-    qmakeFlags="$qmakeFlags QT_HOST_DATA=$out"
-  '';
+  qmakeFlags = [ "MKSPECS_INSTALL_DIR=${placeholder "out"}/mkspecs" ];
```

Comment:
the `QT_HOST_DATA` hack is not needed anymore?

Reply:
> the `QT_HOST_DATA` hack is not needed anymore?

It is not needed anymore.